pkg/reconciler/mtnamespace/resources/broker.go

```diff
@@ -31,6 +31,7 @@ const (
 func MakeBroker(ns *corev1.Namespace) *v1beta1.Broker {
 	return &v1beta1.Broker{
 		ObjectMeta: metav1.ObjectMeta{
+			Annotations: map[string]string{"eventing.knative.dev/broker.class": "MTChannelBasedBroker"},
```

Comment:
There are some constants available for both of these hardcoded strings if you want to change them.

Reply:
https://github.com/knative/eventing/issues/3512